src/java.base/share/classes/javax/crypto/NullCipher.java

```diff
@@ -26,7 +26,7 @@
 package javax.crypto;
 
 /**
- * The NullCipher class is a class that provides an
+ * The <code>NullCipher</code> class is a class that provides an
```

Comment:
Use `{@code }` for all of the javadoc in this class?

Reply:
using `{@code }`